test/ssr/ssrReactive.spec.ts

```diff
@@ -57,6 +57,13 @@ describe('SSR Reactive', () => {
     expect(isRaw(state)).toBe(false)
   })
 
+  it('should work on objects sets with set()', () => {
+    const state = ref<any>({})
+    set(state.value, 'a', {})
```

Comment:
```suggestion
    set(state.value, 'a', {})
    set(state.value, 'a', {})
```

This seems to make a difference